content/api/tracing/send_service.md

```diff
@@ -8,6 +8,6 @@ external_redirect: /api/#send-services
 ## Send services
 ##### Arguments
 
-*   **`service`** - _Required._The service name as a dictionary key, [Learn more](/tracing/terminology).
+*   **`service`** - _Required._The service name as a dictionary key, [Learn more](/tracing/visualization/service).
```

Comment:
Should there be an extra space after `_Required._`?

>*    **`service`** - _Required._ The service name as a dictionary key, [Learn more](/tracing/visualization/service).

Reply:
👌